examples/evaluation_and_profiling/swe_bench/src/nat_swe_bench/predictors/predict_iterative/predict_iterative.py

```
@@ -0,0 +1,505 @@
+            except TerminatingException as e:
+                # Terminal errors: add error message and return
+                self.add_message("user", str(e))
+                return type(e).__name__, str(e)
+
+    async def _query_llm(self) -> str:
+        """Query LLM and return response content."""
+        try:
+            response = await self.llm.ainvoke(self.messages)                    
+            content = response.content if hasattr(response, 'content') else str(response)
+            self.add_message("assistant", content)
+            return content
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)